Sources/FoundationExtensions.swift

```diff
@@ -29,7 +29,7 @@ extension Reactive where Base: NotificationCenter {
 	/// - note: The signal does not terminate naturally. Observers must be
 	///         explicitly disposed to avoid leaks.
 	public func notifications(forName name: Notification.Name?, object: AnyObject? = nil) -> Signal<Notification, NoError> {
-		return Signal { [base = self.base] observer in
+		return Signal(strategy: .inherit) { [base = self.base] observer in
```

Comment:
Sometimes I do wonder if `FlattenStrategy` should be `FlatteningStrategy` instead...